nae.json

```diff
@@ -519,12 +519,6 @@
             }
         },
         "demo-single-public-task": {
-            "layout": {
```

Comment:
why is the layout information removed?

Reply:
because sometimes it makes strange things when you want to generate new schematics and the layout is there and the component already exists